.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: node_js
 node_js:
- - "stable"
+  - "11.10.1"
```

Comment:
I'm curious about this change. Was there a particular reason?

Reply:
There was an error with Travis when it tried to run the test, and after researching about the problem. I found a solution to downgrade the node version from `11.11.1` to `11.10.1`. It turned out Travis had a problem with this node version when It came to testing react components